docs/src/test-assertions-js.md

```diff
@@ -19,10 +19,25 @@ expect(value).not.toEqual(0);
 await expect(locator).not.toContainText("some text");
 ```
 
-You can also specify a custom error message as a second argument to the `expect` function, for example:
+By default, failed assertion will terminate test execution. Playwright also
```

Comment:
I now think that we need a separate #h3 section, so that I see `expect.soft()` and `custom error message` in the right sidebar. WDYT?

Reply:
I'm happy to iterate on the docs in the follow-up!